app/controllers/reports_controller.rb

```diff
@@ -0,0 +1,8 @@
+class ReportsController < ApplicationController
+  def mfp
```

Comment:
Is there a more descriptive name for this? I know it's used internally but might be worth exploding out of the acronym.